runelite-client/src/main/java/net/runelite/client/plugins/party/PartyPanel.java

```diff
@@ -159,6 +159,7 @@ class PartyPanel extends PluginPanel
 					null,
 					null,
 					"");
+				s = s.toLowerCase()
 
 				if (s == null)
```

Comment:
?

Reply:
> ?

Haha mostly writing python right now. Missing semicolon is embarrasing.

Reply:
I was trying to identify the null pointer check after the dereference

Reply:
Ahhh, ya I hadn't made that connnection.